scripts/upload_gatk_docs/upload_gatk_tools_docs.py

```diff
@@ -0,0 +1,238 @@
+                    if '?>' not in line and '<h1' not in line and ending[0] not in line:
+                        if 'class="accordion-body collapse"' in line:
+                            line = line.replace('class="accordion-body collapse"', '')
+
+                        # Add clean HTML to file
+                        clean_lines.append(line)
+
+                    # Starts at the beginning of the end that need to be cut and skips line until the end of the cut
+                    if ending[0] in line:
+                        # Discard all lines up to and including the stop marker
+                        while ending[1] not in line:
+                            line = next(lines)
+                        line = next(lines)
+            except StopIteration:
+                # Create the dictionary of updated gatkdoc
+                article_dict[copied_file] = GatkDocs(title=article_title,
+                                                     file_name=copied_file,
+                                                     local_path=full_path)
+
+        # Write to a clean gatk doc
+        with open(full_path, 'w') as clean_file:
+            clean_file_data = "".join(clean_lines)
+            clean_file_soup = bs(clean_file_data)
+            format_html = clean_file_soup.prettify()
+            clean_file.write(format_html)
+
+    # Output files that was updated
+    print(str(len(copied_files)) + ' files copied and cleaned up')
+
+    # Return GATK Doc Dict
+    return article_dict
+
+
+def post_section(gatk_version, username, token, headers):
+    """Get the url to post an article to the zendesk folder for this gatk release."""
+    url = 'https://gatk.zendesk.com/api/v2/help_center/en-us/sections.json'
+    response = requests.get(url, auth=(username + '/token', token))
+    response = response.json()
+
+    section_exists = False
+    for section in response['sections']:  # look for an existing folder with this gatk version name
+        if section['name'] == gatk_version:
+            url = section['url']
+            section_exists = True
+    if not section_exists:  # if it doesn't already exist, create a folder for this gatk version in Tool Index
+        url_Tool_Index = 'https://gatk.zendesk.com/api/v2/help_center/en-us/categories/360002369672/sections.json'
+
+        json_file = {"section": {"position": 2,
+                                 "sorting": "title",  # will sort contents alphabetically by article title
+                                 "locale": "en-us",
+                                 "name": gatk_version,
+                                 "description": "Tool documentation for GATK release " + gatk_version
+                                 }}
+
+        data = json.dumps(json_file)
+
+        try:
+            # create the folder
+            response = requests.post(url_Tool_Index, headers=headers, data=data, auth=(username + '/token', token))
+        except: 
+            exit("ERROR: Can't make section")
```

Comment:
this error message isn't very descriptive - could you show the error message / actually raise the error instead?